src/main/java/frc/robot/RobotContainer.java

```diff
@@ -17,11 +15,11 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
   // Replace with CommandPS4Controller or CommandJoystick if needed
+  @SuppressWarnings("unused")
   private final CommandXboxController m_driverController =
-      new CommandXboxController(Constants.primaryController);
+      new CommandXboxController(0);
```

Comment:
Same question as above - why remove the constant?

Reply:
Yeah I got rid of it because it didn't match the constants organization, and just forgot to add it back properly. Will commit a fix.